server/routes/postRoutes.js

```diff
@@ -1,11 +1,12 @@
 import express from 'express';
-import { post, get } from '../controllers/postController.js';
+import { post, get, Delete } from '../controllers/postController.js';
 import protect from '../middleware/authMiddleware.js';
 
 
 const router = express.Router();
 
 router.post('/', protect, post);
 router.get('/', protect, get);
+router.delete('/', protect, Delete);
```

Comment:
It would be better to name these more precisely,
getPosts
createPost
deletePost.. 
do you catch my drift ?